dev/compose-mssql.yaml

```diff
@@ -17,4 +17,4 @@ services:
 
 networks:
   service:
-    external: true
\ No newline at end of file
+    external: true
```

Comment:
what was changed here?

Reply:
Prevent compose to create a new network. Use the existing one instead.